rpcs3/Emu/RSX/Overlays/overlays.cpp

```diff
@@ -143,7 +143,7 @@ std::u16string ascii8_to_utf16(const std::string& ascii_string)
 
 	for (const auto& code : ascii_string)
 	{
-		out.push_back(code > 0xFF ? '#' : (char16_t)code);
```

Comment:
this one is intentional

Reply:
I noticed immediately, guess the zombie comment stuck around.

Reply:
ah i see. so it was a copy pasta after all